<launch>

  <!-- Nodes for traversability: we need elevation mapping and the message interceptor node -->
  <node pkg="elevation_mapping" type="elevation_mapping" name="elevation_mapping" output="screen">
    <rosparam command="load" file="$(find smb_planner_common)/cfg/elevation_mapping_smb.yaml" />
    <rosparam command="load" file="$(find elevation_mapping)/config/sensor_processors/realsense_d435.yaml" />
  </node>
  
  <node pkg="smb_planner_common" type="smb_pose_interceptor_node" name="smb_pose_interceptor" output="screen" >
    <remap from="/base_pose_measured" to="/base_pose_measured_disabled" />
  </node>
  
  <!-- Launch visualizations for the resulting elevation map -->
  <include file="$(find elevation_mapping_demos)/launch/visualization.launch" />
  
</launch>
